uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/JobRequestHelper.java

```diff
@@ -462,22 +451,21 @@ public JobResult removeOrganizationalUnit(final String jobId,
     }
 
     public JobResult createOrganizationalUnit(final String jobId,
-                                              final String organizationalUnitName,
-                                              final String organizationalUnitOwner,
-                                              final String defaultGroupId,
-                                              final List<String> repositoryNameList) {
+                                              final String spaceName,
```

Comment:
I am not sure if the name of the method is still actual. Shouldn't it be called createSpace?

Reply:
I think it should, yes. Renamed.